src/definition/formatter/number.js

```diff
@@ -1,16 +1,23 @@
-const numeral = require('numeral');
+import numeral from 'numeral';
 
 const DEFAULT_FORMAT = '0,0';
 
-module.exports = {
+//TODO change numeral lib and regroup initializers
+export function language(key, conf) {
```

Comment:
I agree with you.
But we discuss with @pierr. He prefers to expose the same method.